backend/src/main/java/com/github/licetrosales/backend/controller/RecipeController.java

```diff
@@ -37,4 +36,22 @@ void deleteRecipe(@PathVariable String id) {
         recipeService.delete(id);
     }
 
+    @PutMapping(path = "/recipes/{id}")
+    Recipe updateRecipe(@PathVariable String id, @RequestBody RecipeDTO recipeToUpdateWithoutId) {
+        Recipe recipeToUpdateWithId = new Recipe(id,
+                recipeToUpdateWithoutId.name(),
+                recipeToUpdateWithoutId.mealType(),
+                recipeToUpdateWithoutId.source(),
+                recipeToUpdateWithoutId.image(),
+                recipeToUpdateWithoutId.ingredients(),
+                recipeToUpdateWithoutId.prepTime(),
+                recipeToUpdateWithoutId.preparation(),
+                recipeToUpdateWithoutId.portions(),
+                recipeToUpdateWithoutId.favorite(),
+                recipeToUpdateWithoutId.dishTypeCategory(),
+                recipeToUpdateWithoutId.recipeCategory(),
+                recipeToUpdateWithoutId.menuCategory(),
+                recipeToUpdateWithoutId.garnish());
```

Comment:
Es wäre cool, wenn der Controller mögichst schlank bleibt und diese Logik im Service passiert.
D.h. updateRecipe erhält zwei Parameter: id und RecipeDTO und baut das Recipe dann in der Methode